module/data/item/equipment.mjs

```diff
@@ -132,4 +145,22 @@ export default class EquipmentData extends SystemDataModel.mixin(
     return this.armor.type === "vehicle";
   }
 
+  /* -------------------------------------------- */
+
+  /**
+   * The proficiency multiplier for this item.
+   * @returns {number}
+   * @protected
+   */
+  get _proficiencyMultiplier() {
```

Comment:
Perhpas this is worth considering as "public API" instead of `@protected`? Could be useful for various things.